server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -89,20 +89,24 @@
     private String username;
```

Comment:
Cucumber tests? :trollface:

Reply:
This integration test is supposed to be implemented in https://ci.linagora.com/linagora/lgs/openpaas/james/issues/362

Reply:
Please drop these tests! They are not part of that ticket, you should provide "real" unit tests for your listener!